client/src/screens/createTraining/controller.js

```diff
@@ -3,9 +3,30 @@ export const controller = (serviceLocator) => {
   const { trainingsApi, navigationApi } = serviceLocator.getAPIs();
 
   const getData = () => getStoreData(controller.storeDataAccessors);
+  const createSetsPreview = (sets) => {
+    let setsPreview = '';
+
+    sets.forEach((set, index) => {
+      setsPreview += `${set.repetitions}x${set.weight}kg`;
+      if (index < (sets.length - 1)) setsPreview += ' - ';
+    });
+
+    return setsPreview;
+  };
```

Comment:
you already had the same code. looks like a code duplication.
if this is true, then reuse the code.

Reply:
here https://github.com/vincefliek/YourGym/blob/main/client/src/screens/training/controller.js#L9-L18